### PR TITLE
create dy-fileserver service for testing

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- created a set of dynamic services based on nginx
- serve a hello-world.txt file
- 1 classical dynamic service
- 1 modern dynamic service to be used with the dynamic sidecar
- 1 modern dynamic service to be used with the dynamic sidecar and the compose specifications

- removed httpbin service as not following any of the conventions.

## Checklist

- [ ] I think the code is well written
- [ ] I updated the table of contents (make toc)
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/dy-static-file-server/src/dy_static_file_server/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+add source code here
```
Comment (pcrespov) [inline services/dy-static-file-server/src/dy_static_file_server/.gitkeep:1]: No need for the .gitkeep files

File: services/dy-static-file-server/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+version: '3.7'
+services:
+  dy-static-file-server:
+    init: true
+    image: ${DOCKER_REGISTRY}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
```
Comment (pcrespov) [inline services/dy-static-file-server/docker-compose.yml:5]: missed name image?

**Review by pcrespov — APPROVED** [review]
- three nginx services from the same ``Dockerfile``
- the only difference between them is the names

Some questions:
- does /shoudn't the ``integration-version`` have any role now with dynamic-services?
- I assume you need these services as dummies to test the dynamic sidecar.

File: services/dy-static-file-server/src/dy_static_file_server/.gitkeep (hunk 1 above)
Comment (sanderegg) [reply services/dy-static-file-server/src/dy_static_file_server/.gitkeep:1]: I'd like to keep it, in case later on we modify that service

**Comment by sanderegg** [comment]
> * three nginx services from the same `Dockerfile`
> * the only difference between them is the names
No also the service labels are different (the typical dynamic-sidecar required labels are on the "modern" services). and the legacy service also handles the SIMCORE_NODE_BASEPATH since it needs to serve the files on /x/{node_uuid}

> 
> Some questions:
> 
> * does /shoudn't the `integration-version` have any role now with dynamic-services?
Yes and no. The integration version could be used. but I think in this case we are talking of a dynamic-service-specific change. so not sure if using the integration version will not bring more confusion. With the current dynamic-service PR the required fields are:
```
simcore.service.boot-mode: dynamic-sidecar # I would consider removing this label
simcore.service.compose-spec: ${DOCKER_COMPOSE_SPECIFICATION} # this is an optional label that contain the docker-compose if available on how to start dependencies of the service (if any). can be empty or not present.
simcore.service.container-http-entrypoint: httpbin-docker-compose-spec # in case the compose-spec field is filled, this would define the main service. also not sure if that is really needed. @GitHK might have to comment this one.
simcore.service.paths-mapping: '{"outputs_path": "/www/outputs", "inputs_path":
          "/www/inputs"}' # defines the INPUT/OUTPUT folders needed by the dynamic services
```
> * I assume you need these services as dummies to test the dynamic sidecar.
yes. the nginx image is 22MB large, so it should be not too bad to pull for testing.

File: services/dy-static-file-server/dynamic-sidecar-compose-specs.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  dy-static-file-server-dynamic-sidecar-compose-spec:
+    init: true
+    image: ${DOCKER_REGISTRY}/${DOCKER_IMAGE_NAME}-dynamic-sidecar-compose-spec:${DOCKER_IMAGE_TAG}
+    environment:
+      - MOCK_VALUE=TheMockedValue
+
+  some-side-service:
+    init: true
+    image: busybox:latest
+    cmd: top
```
Comment (GitHK) [inline services/dy-static-file-server/dynamic-sidecar-compose-specs.yml:12]: 👍 cool

**Pull request merged**